ADBench/plot_graphs.py

```diff
@@ -212,21 +234,26 @@ def print_messages():
 
     if do_help:
         ref_msg = f'''
-This script produces graphs that visualize benchmark.
+This script produces graphs that visualize benchmark. Also, it saves time info
+that is used for graph creating.
 CMD arguments:
     --save
-            if specified then script saves produced graphs to
+            if specified then the script saves produced graphs to
             {static_out_dir}
 
     --plotly
-            if specified then script saves graphs in plotly format to
+            if specified then the script saves graphs in plotly format to
```

Comment:
Different people have different preferences regarding this, but FYI my personal preference is if typo-changes are submitted in separate PRs.  The difficulty for me of reviewing a PR is O(n^2) in the number of lines changed (I think about all possible new interactions between pairs of lines changed) so submitting independent changes in independent PRs picks up a significant performance improvement.

Reply:
(This is an observation only, not a request for a change to this PR!)